DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -627,34 +614,21 @@ private void retrieveExhibitionMapsInternal() {
         }
 
         if (favoredSessions == null) {
-            favoredSessions = internalRetrieveFavoredSessions();
-        }
-
-        return favoredSessions;
-    }
-
-    @Override
-    public ObservableList<Session> retrieveScheduledSessions() {
-        if (!isAuthenticated()) {
-            throw new IllegalStateException("An authenticated user must be available when calling this method.");
-        }
-
-        if (scheduledSessions == null) {
             try {
                 DevoxxNotifications notifications = Injector.instantiateModelOrService(DevoxxNotifications.class);
                 // stop recreating notifications, after the list of scheduled sessions is fully retrieved
-                scheduledSessions = internalRetrieveScheduledSessions(notifications::preloadingScheduledSessionsDone);
+                favoredSessions = internalRetrieveFavoredSessions(notifications::preloadingScheduledSessionsDone);
```

Comment:
Rename `preloadingScheduledSessionsDone` to `preloadingFavoriteSessionsDone`?

Reply:
The reason I haven't changed the name of this and subsequent methods mentioned below is because these methods are still scheduling notifications for favorite sessions.